pleskdistup/common/src/files.py

```diff
@@ -59,25 +59,30 @@ def get_last_lines(filename: PathType, n: int) -> typing.List[str]:
         return f.readlines()[-n:]
 
 
-def backup_file(filename: str) -> None:
+def backup_file(filename: str, ext: str = ".bak") -> None:
     if os.path.exists(filename):
```

Comment:
I think we should check if the backup file already exists and skip copying it if it does. This prevents overwriting the backup file if it has already been created and the .list file has been changed

Reply:
But this will risk when user has a '.bak' long long times ago.

Reply:
Let's then change it to a less common suffix like "conversion.back". I want to ensure we don't get overwhelmed by various suffixes for each case.